### PR TITLE
feat: Allow passing any Jest CLI options to neutrino-jest test task

### LINKED ISSUE #287: Add `findRelatedTests` CLI option to jest preset
It looks like Jest preset CLI options are missing `findRelatedTests` options that is useful with tools like [lint-staged](https://github.com/okonet/lint-staged)

https://github.com/mozilla-neutrino/neutrino-dev/blob/master/packages/neutrino-preset-jest/src/index.js#L97

### PR DESCRIPTION
**Pull request by okonet**
Closes #287 

At the moment it's not possible to run `neutrino test` with additional options Jest provides. This includes `--findRelatedTests` or `--onlyChanged` which can be very useful. 

This PR implements an ability to pass arbitrary options to the Jest CLI tool.

### CODE REVIEW

File: packages/neutrino-preset-jest/src/index.js [hunk 1]
```diff
@@ -92,13 +93,26 @@ module.exports = (neutrino, opts = {}) => {
...
 
     return new Promise((resolve, reject) => {
+      // We need to parse argv separately in order to identify files
+      // and jest-related options since root neutrino does not know about
+      // jest options and will provide wrong/incomplete `args.files`
+      const jestArgs = yargs
+        .command('test [files..]', 'Run tests', jestOptions)
+        .argv;
       const configFile = join(tmpdir(), 'config.json');
       const options = normalizeJestOptions(opts, neutrino, args);
```
Comment (eliperelman) [inline packages/neutrino-preset-jest/src/index.js]: Since `normalizeJestOptions` no longer accepts `args`, we can remove them from being passed here.

Comment (okonet) [reply packages/neutrino-preset-jest/src/index.js]: Good catch! Completely forgot about it here!

Comment (okonet) [reply packages/neutrino-preset-jest/src/index.js]: Done!

**Comment by eliperelman** [comment]
@okonet one more request: would you mind adding some small docs to `packages/neutrino-preset-jest/README.md` and `docs/presets/neutrino-preset-jest/README.md` about using this? The files are pretty much the same, should just copy what you write from one to the other.

**Comment by okonet** [comment]
@eliperelman done!

**Comment by eliperelman** [comment]
@okonet you're my hero

**Comment by eliperelman** [comment]
Released in v6.2.0.

**Pull request merged**